README.md

```diff
@@ -1 +1,24 @@
-# Recipe_App
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
```

Comment:
- [ ] Please update your readme file to match your current project. You can follow this[ template.](https://github.com/microverseinc/readme-template/blob/master/README.md). It will make your project  more professional